src/Core/src/Hosting/EssentialsMauiAppBuilderExtensions.cs

```diff
@@ -158,10 +166,12 @@ void HandleOnAppAction(object? sender, AppActionEventArgs e)
 
 		class EssentialsBuilder : IEssentialsBuilder
 		{
-			internal readonly List<AppAction> AppActions = new List<AppAction>();
+			private List<AppAction>? _appActions;
```

Comment:
we don't really use private in our code style , not important, but just a reminder for future pr's